ansible/library/port_alias.py

```diff
@@ -112,7 +112,7 @@ def get_portmap(self):
                     aliases.append(alias)
                     portmap[name] = alias
                     aliasmap[alias] = name
-                    if speed_index != -1:
+                    if (speed_index != -1) and (len(mapping) > speed_index):
```

Comment:
```python
alias_index = -1
speed_index = -1
title = None
while len(lines) != 0:
...
if re.match('^\w', line):
    mapping = line.split()
    if title and len(mapping) == len(title):
        ...
        if speed_index != -1:
            portspeed[alias] = mapping[speed_index]
```

Reply:
In the code to parse port_config.ini, seems that even len(mapping) != len(title), in some cases the port_config.ini could still be valid.  https://github.com/Azure/sonic-platform-common/blob/master/sonic_sfp/sfputilbase.py#L400